### PR TITLE
Hover changes

### LINKED ISSUE #964: Hover styles of back and close actions
## Describe the bug
In the pop-up of **report this content**, option "other", the `back` and `close` actions have incorrect styles during hover state.

## Expected behavior
In hover state, it should happen the following:
1. The back action should change the text color from `Gray` to `DarkGray`.
2. The fill color of the cross icon should change `Gray` to `DarkGray`.
See the [Mockup S3 in Figma](https://www.figma.com/file/KEXxItd8EloJGbXswuUDqq/Mockup---S3-(License-Language-Changes)?node-id=199%3A354&viewport=968%2C-5669%2C1.085633635520935).

## Screenshots
Screen Shot 2020-05-14 at 15 34 57
Screen Shot 2020-05-14 at 15 34 53

### PR DESCRIPTION
**Pull request by akanksha-v**
## Fixes

Fixes #964

## Description
1. The back action is now changing the text color from Grey to dark grey on hovering.
2. The fill color of the cross icon is also changing from Grey to dark grey on hovering.

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: src/components/ContentReport/ContentReportForm.vue [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <template>
   <div class="padding-normal is-clearfix report-form">
     <button class="button close-button is-text tiny is-pulled-right is-block has-text-grey-light"
-            @click="closeForm()">
-      <i class="icon cross"></i>
+        id="cross-icon"  @click="closeForm()">
```
Comment (brenoferreira) [inline src/components/ContentReport/ContentReportForm.vue:4]: please don't style things with element ID

File: src/components/ContentReport/ContentReportForm.vue [hunk 2]
```diff
@@ -53,8 +53,8 @@
                 placeholder="Issue description required (with at least 20 characters)" />
       <div>
         <button class="button other-back-button is-text tiny margin-top-normal has-text-grey"
-                @click="onBackClick()">
-          <span><i class="icon chevron-left margin-right-small"></i> Back</span>
+              @click="onBackClick()">
+          <span id="back-button"><i class="icon chevron-left margin-right-small"></i> Back</span>
```
Comment (brenoferreira) [inline src/components/ContentReport/ContentReportForm.vue:57]: same here

File: src/components/ContentReport/ContentReportForm.vue [hunk 3]
```diff
@@ -139,4 +137,10 @@ export default {
   font-size: 13px;
   font-family: Source Sans Pro;
 }
+#back-button:hover{
+  color:rgb(73, 73, 73);
+}
+#cross-icon:hover{
+  background-color: grey;
```
Comment (brenoferreira) [inline src/components/ContentReport/ContentReportForm.vue:144]: the issue says the text color needs to change on hover, not the background color

**Comment by zackkrida** [comment]
Hi @akanksha-v, if you would like to continue this PR please address the following items:

1. Remove added `id` attributes
2. Remove the background color on the "x" (close) icon
3. Please make the "x" (close) icon and the back action text use `$color-dark-gray` instead of hand-written css colors

Thank you! If you need help on any of the above feel free to ask.

**Comment by akanksha-v** [comment]
@zackkrida  sorry for the inconvenience but I am really confused for what should I do for hover property as I am unable to find any in Design library. Could you please help me with it

**Comment by zackkrida** [comment]
Hi @akanksha-v after reviewing this more, I realize the problem is actually in https://github.com/creativecommons/vocabulary, so I will have the changes made there instead.

Thank you for looking at this!

**Pull request closed without merging**